internal/eval/commands.go

```diff
@@ -18,6 +18,7 @@ type KeySpecs struct {
 
 var (
 	DiceCmds = map[string]DiceCmdMeta{}
+	CommandSubCommands []string
```

Comment:
Can this be moved info the diceCmdMetadata struct for COMMAND?

Reply:
Yes, that sounds like a good idea. This  will allow us to easily add sub-commands for other commands too.